src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -1,4 +1,35 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Random;
+
+@RestController
+@RequestMapping("/users")
 public class UserController {
+    private long key;
+
+    private UserMapper userMapper;
+
+    @Autowired
+    public UserController(UserMapper userMapper) {
+        this.userMapper = userMapper;
+    }
+
+    @PostMapping("createUser")
+    public User createUser(@RequestBody UserDto userDto) {
+        return userMapper.mapToUser(userDto);
+    }
+
+    @GetMapping("banUser")
+    public void banUser(Long userId) {
+
+    }
+
+    @GetMapping("generateKey")
+    public Long generateKey() {
```

Comment:
ten klucz specjalnie generujemy w oderwaniu od obiektu użytkownika?

Reply:
to miała być taka wersja tymczasowa, jak widać niezbyt udana ;] w nowej zrobiłem poprzez konkatenacje losowego klucza i id użytkownika